Firmware/messages.c

```diff
@@ -52,7 +54,7 @@ const char MSG_HEATING_COMPLETE[] PROGMEM_I1 = ISTR("Heating done."); ////c=20
 const char MSG_HOMEYZ[] PROGMEM_I1 = ISTR("Calibrate Z"); ////c=18
 const char MSG_ITERATION[] PROGMEM_I1 = ISTR("Iteration"); ////c=12
 const char MSG_CHOOSE_EXTRUDER[] PROGMEM_I1 = ISTR("Choose extruder:"); ////c=20
```

Comment:
Would it make sense to change `choose extruder` to `select extruder` as well? Just to make it more consistent?